api-tests/features/return-codes.feature

```diff
@@ -0,0 +1,174 @@
+Feature: Return exit codes
+  Rule: no return codes
+    @Build
+    Scenario:  Successful journey with identity - no CIs
+      When I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'appTriage' event
+      Then I get a 'dcmaw' CRI response
+      When I submit 'kenneth-passport-valid' details to the CRI stub
+      Then I get a 'page-dcmaw-success' page response
+      When I submit a 'next' event
+      Then I get an 'address' CRI response
+      When I submit 'kenneth-current' details to the CRI stub
+      Then I get a 'fraud' CRI response
+      When I submit 'kenneth-score-2' details to the CRI stub
+      Then I get a 'page-ipv-success' page response
+      When I submit a 'next' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P2' identity
+      And I don't get any return codes
+
+  Rule: non-ci-breaching code returned
+    Scenario: Failed identity journey with no CI - user doesn't hold appropriate documents
+      Given I start a new 'medium-confidence' journey
+      Then I get a 'page-ipv-identity-document-start' page response
+      When I submit an 'end' event
+      Then I get a 'page-ipv-identity-postoffice-start' page response
+      When I submit an 'end' event
+      Then I get a 'pyi-escape' page response
+      When I submit a 'end' event
+      Then I get an OAuth response
+      When I use the OAuth response to get my identity
+      Then I get a 'P0' identity
+      And I get 'non-ci-breaching' return code
+
+    Rule: Failed to complete journey
+      Background: Start alternate doc journey with no CI
```

Comment:
I don't think this is an 'alternate doc' journey? Looks like thin file + failure to use dcmaw/f2f